src/status_im/utils/db.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im.utils.db
+  (:require [clojure.string :as string]
+            [cljs.spec.alpha :as spec]
+            [status-im.js-dependencies :as dependencies]))
+
+(defn address? [s]
+  (.isAddress dependencies/Web3.prototype s))
+
+(defn hex-string? [s]
```

Comment:
Those helper methods look generic? it would be helpful to have them defined in appropriate `utils` ns so that they can be found/reused.
Currently our `utils` ns is kind of messy.

Maybe also worth adding tests/doc?